src/ui/SimpleFilter/SimpleFilter.ts

```diff
@@ -458,6 +470,21 @@ export class SimpleFilter extends Component {
     return this.circleElement.el;
   }
 
+  public buildClearElement(): HTMLElement {
```

Comment:
I find methods that set class-wide properties and that also have a return type to be a little messy. It's like combining a getter and a setter into one function. I prefer to use or the other.

Also, does this method need to be `public`?
e.g.

```
private buildClearButton(): Dom {
     const button = ...
     return button;
}
```
or
```
private initClearButton() {
    const button = ...
    this.clearButton = button;
}
```